kfs-core/src/main/resources/edu/arizona/kfs/workflow/DisbursementVoucherDocument.xml

```diff
@@ -0,0 +1,133 @@
+            </branch>
+            <join name="JoinPayeeIsPurchaseOrderVendor" />
+          </split>
+          <role name="Account" nextNode="AccountingOrganizationHierarchy" />
+          <role name="AccountingOrganizationHierarchy" nextNode="ObjectSubTypeCode" />
+          <role name="ObjectSubTypeCode" nextNode="RequiresTaxReview" />
+          <split name="RequiresTaxReview" nextNode="Award">
+            <branch name="True">
+              <role name="Tax" nextNode="JoinRequiresTaxReview" />
+            </branch>
+            <branch name="False">
+              <simple name="NoOpTaxReview" nextNode="JoinRequiresTaxReview" />
+            </branch>
+            <join name="JoinRequiresTaxReview" />
+          </split>
+          <role name="Award" nextNode="RequiresTravelReview" />
+          <split name="RequiresTravelReview" nextNode="Campus">
+            <branch name="True">
+              <role name="Travel" nextNode="JoinRequiresTravelReview" />
+            </branch>
+            <branch name="False">
+              <simple name="NoOpTravelReview" nextNode="JoinRequiresTravelReview" />
+            </branch>
+            <join name="JoinRequiresTravelReview" />
+          </split>
+          <role name="Campus" nextNode="PaymentMethod" />
+          <role name="PaymentMethod" nextNode="RequiresSeparationOfDutiesReview" />
+          <split name="RequiresSeparationOfDutiesReview">
+            <branch name="True">
+              <role name="SeparationOfDuties" nextNode="Join" />
+            </branch>
+            <branch name="False">
+              <simple name="NoOp" nextNode="Join" />
+            </branch>
+            <join name="Join" />
+          </split>
+        </routePath>
+      </routePaths>
+      <routeNodes>
+        <start name="AdHoc" />
+        <split name="PayeeIsPurchaseOrderVendor">
+          <type>org.kuali.kfs.sys.document.workflow.SimpleBooleanSplitNode</type>
+        </split>
+        <role name="Purchasing">
+          <qualifierResolverClass>org.kuali.rice.krad.workflow.attribute.DataDictionaryQualifierResolver</qualifierResolverClass>
+          <activationType>P</activationType>
+        </role>
+        <join name="JoinPayeeIsPurchaseOrderVendor" />
+        <simple name="NoOpPayee">
+          <type>org.kuali.rice.kew.engine.node.NoOpNode</type>
+        </simple>
+        <role name="Account">
+          <qualifierResolverClass>org.kuali.rice.krad.workflow.attribute.DataDictionaryQualifierResolver</qualifierResolverClass>
+          <activationType>P</activationType>
+        </role>
+        <role name="AccountingOrganizationHierarchy">
+          <qualifierResolverClass>org.kuali.rice.krad.workflow.attribute.DataDictionaryQualifierResolver</qualifierResolverClass>
+          <activationType>P</activationType>
+        </role>
+        <role name="ObjectSubTypeCode">
+	      <qualifierResolverClass>org.kuali.rice.kns.workflow.attribute.DataDictionaryQualifierResolver</qualifierResolverClass>
```

Comment:
should be:
org.kuali.rice.krad.workflow.attribute.DataDictionaryQualifierResolver